### ISSUE TITLE
v.enum.map is not a function

### ISSUE THREAD

**Issue by kpietrzak-web** [Manweill/swagger-axios-codegen#19]
Error appears in swagger-axios-codegen/dist/definitionCodegen/propTrueType.js:50:34

If enum is swagger output is defined as object instead of array codegen shows this message.

Solution is:
 ```
       if(typeof v.enum === "object") {
            v.enum = Object.values(v.enum);
        }
```

**Comment by Manweill** [comment]
@kpietrzak-web 
Wecome PR.

**Comment by Manweill** [comment]
@kpietrzak-web 
Hi, can you give me a swagger of you

**Comment by Manweill** [comment]
fix in v0.5.3
